core/src/test/kotlin/dev/hotwire/core/turbo/config/PathConfigurationRepositoryTest.kt

```diff
@@ -53,6 +53,20 @@ class PathConfigurationRepositoryTest : BaseRepositoryTest() {
         assertThat(config?.rules?.size).isEqualTo(10)
     }
 
+    @Test
+    fun getMalformedRemoteConfiguration() {
+        enqueueResponse("malformed-body.json")
+        val loader = PathConfigurationLoader(context)
+
+        runBlocking {
+            launch(Dispatchers.Main) {
+                val json = repository.getRemoteConfiguration(baseUrl())
+                val pathConfiguration = json?.let { loader.load(it) }
+                assertThat(pathConfiguration).isNull()
+            }
+        }
+    }
```

Comment:
The more appropriate place for this test is in `PathConfigurationTest`, since parsing exceptions are caught at the `PathConfigurationLoader` layer, not the repository layer.